server/autolinkplugin/plugin_test.go

```diff
@@ -55,6 +57,259 @@ func TestPlugin(t *testing.T) {
 	assert.Equal(t, "Welcome to [Mattermost](https://mattermost.com)!", rpost.Message)
 }
 
+type SuiteAuthorization struct {
```

Comment:
I took the liberty of not being consistent with other tests in this file, as using testing suite approach IMO is cleaner and we are already using testing suites in e.g. `mattermost-server` repo.